### PR TITLE
Code Jam 8 - Initial PR for Announcement

### PR DESCRIPTION
**Pull request by janine9vn**
This Pull Request is just the basics required for the Code Jam announcement. It does the following things:
1. Add in basic assets for the event
2. Create the CJ8 pages
3. Update the main events page and the main code jam page

There will be a second PR made this Sunday that will have more of the details and include the Qualifier release.

### CODE REVIEW

File: pydis_site/templates/events/pages/code-jams/8/_index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        Your program must use the specified technology/framework and incorporate the theme chosen by the server.
+    </p>
+    <p>
+        After the 7 days is complete, your team has 2 days to finish documentation and create a video presentation showcasing
+        and walking through the program that your team has created. More details and specifics of this will be released within the next 2 weeks. 
+    </p>
+
+    <h3 id="important-dates"><a href="#important-dates">Important Dates</a></h3>
+    <ul>
+        <li>Monday, June 14 - Form to submit theme suggestions opens</li>
```
Comment (jb3) [inline pydis_site/templates/events/pages/code-jams/8/_index.html]:
```suggestion
        <li>Monday, June 15 - Form to submit theme suggestions opens</li>
```

File: pydis_site/templates/events/pages/code-jams/8/_index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        <li>Wednesday, June 30 - The Qualifier closes</li>
+        <li>Friday, July 9 - Code Jam Begins</li>
+        <li>Friday, July 16 - Coding portion of the jam ends</li>
+        <li>Sunday, July 18 - Code Jam submissions are closed</li>
+        
+    </ul>
+
+    <h3 id="how-to-join"><a href="#how-to-join">How to Join</a></h3>
+    <p>The Qualifier isn't released yet, but to receive the most up-to-date information and to get notified 
+        when the Qualifier released you can join the server: <a href="https://discord.gg/python">discord.gg/python</a>.</p>
```
Comment (Shivansh-007) [inline pydis_site/templates/events/pages/code-jams/8/_index.html]:
```suggestion
        when the Qualifier is released you can join the server: <a href="https://discord.gg/python">discord.gg/python</a>.</p>
```

**Review by jb3 — APPROVED** [review]
Other than linting failure, all looks good, approving.

File: pydis_site/templates/events/pages/code-jams/8/rules.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        <p>
+            Use English as the main language for your project, including names, comments, documentation, and commit messages.
+            The text displayed in your web application should also be in English,
+            although you are allowed to provide the user with options for internationalisation and translation.
+        </p>
+    </li>
+    <li>
+        <p>
+            Your team, once the coding portion of the code jam is complete, must create a video presentation that showcases and explains your final product.
+            This must be in a video format and must be uploaded somewhere for the judges to view (i.e. unlisted youtube video, vimeo, etc.)
```
Comment (Shivansh-007) [inline pydis_site/templates/events/pages/code-jams/8/rules.html]:
```suggestion
            This must be in a video format and must be uploaded somewhere for the judges to view (i.e. unlisted YouTube video, Vimeo, etc.)
```

**Review by jacobmonck — APPROVED** [review]
Looks good to me

**Review by Shivansh-007 — APPROVED** [review]
Wait

**Review by whispersofthedawn — APPROVED** [review]
LGTM

**Pull request merged**